oauth2/src/main/java/com/linecorp/armeria/client/auth/oauth2/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Client-side OAuth 2.0 implementation.
```

Comment:
How about adding a hyperlink to a good resource about OAuth 2.0 for the text `OAuth 2.0`?

Reply:
fixed